force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -299,18 +361,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return The Map of fruits and their quantities.
      */
     public static Map<String, Integer> addItemsToMap() {
-        // Initialize an empty Map
-
-        // Add key-value pairs to the Map
-        // add Apples => 5
-        // add Oranges => 10
-        // add Bananas => 15
-        // add Pears => 20
-        // add Grapes => 25
-
-        return null; // Replace null with the variable you used to store the result
+        // Create new map called fruitOnHand
+        Map<String, Integer> fruitOnHand = new Map<String, Integer>();
+        fruitOnHand.put('Apples', 5);
+        fruitOnHand.put('Oranges', 10);
+        fruitOnHand.put('Bananas', 15);
+        fruitOnHand.put('Pears', 20);
+        fruitOnHand.put('Grapes', 25);
+        return fruitOnHand;  // Passes Debug
```

Comment:
good job. Another technique is to to use the Map constructor 

 return new Map<String, Integer>{
                'Apples' => 5,
                'Oranges' => 10,
                'Bananas' => 15,
                'Pears' => 20,
                'Grapes' => 25
        };